MediaBrowser.Model/Providers/ExternalIdMediaType.cs

```diff
@@ -0,0 +1,77 @@
+namespace MediaBrowser.Model.Providers
+{
+    /// <summary>
+    /// The specific media type of an <see cref="ExternalIdInfo"/>.
+    /// </summary>
+    /// <remarks>
+    /// Client applications may use this as a translation key.
+    /// </remarks>
+    public enum ExternalIdMediaType
```

Comment:
Should we be setting the Enum value here?

Reply:
I've added explicit enumeration values here